netutils/password.py

```diff
@@ -207,9 +207,11 @@ def encrypt_type7(unencrypted_password, salt=None):
     """
     if not salt:
         salt = random.randrange(0, 15)  # nosec
-    encrypted_password = "%02x" % salt  # pylint: disable=consider-using-f-string
+    encrypted_password = "%02x" % salt  # pylint: disable=bad-option-value, consider-using-f-string
```

Comment:
These were passing tests before, why is this needed now? 

Apologies, I know there is some odd packaging things going on, so may be outside of anything you are doing, just trying to understand.

Reply:
Obviously I don't want to go near the `password` file but pylint was complaining. Removed it and seems to be ok.